### ISSUE TITLE
Add support for adjusting visualizations based on available sample metadata

### ISSUE THREAD

**Issue by khughitt** [NCBI-Hackathons/Pharmacogenomics_Prediction_Pipeline_P3#22]
Currently, the visualization part of the pipeline works in a very generic manner and does not attempt to stylize data points based on available metadata.

It may be desirable, however, to allow user's to configure the pipeline so that color, shape, etc. are  mapped to different features of the dataset.

For instance, with RNA-Seq, if there are meaningful ways to categorize the samples, then one could color the points in the PCA plot and add colorbars to the heatmaps to indicate category membership.

**Comment by daler** [comment]
Good idea. I think it would be best to come up with a format that works for the visualization scripts and then we'll worry about how to add to the config later.
